llvm/utils/lit/lit/formats/googletest.py

```diff
@@ -33,6 +33,7 @@ def get_num_tests(self, path, litConfig, localConfig):
             [path, "--gtest_list_tests", "--gtest_filter=-*DISABLED_*"]
         )
         try:
+            localConfig.environment['DYLD_LIBRARY_PATH'] = ''
```

Comment:
Don’t think you meant to commit that.

Reply:
Iiiii did not. I should also figure out one of these days why I need it 😂

Reply:
Done, thanks for noticing that.